P5/Source/Specs/att.formula.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<classSpec xmlns="http://www.tei-c.org/ns/1.0" module="tei" type="atts" ident="att.formula">
+  <desc versionDate="2019-06-25" xml:lang="en">provides attributes for defining a mathematical formula.</desc>
+  <attList xmlns:rng="http://relaxng.org/ns/structure/1.0">
+    <attDef ident="formula" usage="opt">
+      <gloss versionDate="2019-06-25" xml:lang="en">holds a numeric value designated for use in a mathematical calculation.</gloss>
```

Comment:
This `<gloss>` contains a description. I think it is not necessary to provide a `<gloss>` in this case.

Reply:
@martinascholger So I should change this to a `<desc>` as well? But I might need to modify this anyway--if we're following up on @martindholmes 's idea here. I have been wondering if we want to take a closer look at this new attribute class and attribute anyway...

Reply:
Also in att.formula.xml: the second sentence of the `<remarks>` contains a word duplication - from from